configure

```diff
@@ -3966,6 +3971,82 @@ echo "$_freetype2"
 define_in_config_if_yes "$_freetype2" "USE_FREETYPE2"
 
 #
+# Check for iconv
+#
+# TODO: Support prefix for iconv
```

Comment:
Please include the following commits from ResidualVM so that iconv support is in sync between the project:
https://github.com/residualvm/residualvm/commit/33d5705af48106291c92837eb97374fec3c14db1
https://github.com/residualvm/residualvm/commit/8e51ec52ec2c9c86552138f2529dddb9be60ec98